### DISCUSSION TITLE
Single vs multiple drives per node in MinIO cluster

### DISCUSSION THREAD

**Discussion by fpetkovski** [minio/minio#20754] [Q&A]
I've been trying to understand the tradeoff between using a single drive per node, vs using fewer nodes with more drives.
For example, what is the performance tradeoff between a cluster of 16 nodes with 1 drive each, vs one with 8 nodes with 2 drives each?

The documentation also claims that MinIO requires 4 drives to enable erasure coding: https://min.io/docs/minio/linux/reference/minio-server/minio-server.html#minio.server.DIRECTORIES. Are drives from other nodes also considered or do those drives have to be local?

**Comment by klauspost** [comment]
It mainly affects your fault tolerance; but in practice it should be close to the same. For discussions on minio setups, reach out on `sales@min.io`.
